src/components/Section/SectionWide.jsx

```diff
@@ -21,6 +21,7 @@ const useStyles = makeStyles((theme) => ({
     },
     [theme.breakpoints.up("lg")]: {
       height: "400px",
+
```

Comment:
? 😅

Reply:
should be resolved in GitHub, in my VSCode all good..